spark-plugins/docs/Tokenizer-sparkcompute.md

```diff
@@ -0,0 +1,58 @@
+# Tokenizer Spark Compute
+
+
+Description
+-----------
+Tokenization is the process of taking text (such as a sentence)
+and breaking it into individual terms (usually words) on the basis of pattern.
+
+Tokenizer splits data on the basis of specified pattern and emits the output as string array of tokens.
+
+Use Case
+--------
+Tokenize the data on the basis of pattern.
+Tokenized output will be an array of tokens.
```

Comment:
@shashankmoghe please could you update this with the use case? the code looks fine, it is just missing the use case.